dependency_support/load_external.bzl

```diff
@@ -180,3 +180,57 @@ def load_external_repositories():
         strip_prefix = "ac_types-57d89634cb5034a241754f8f5347803213dabfca",
         build_file = "@com_google_xls//dependency_support/com_github_hlslibs_ac_types:bundled.BUILD.bazel",
     )
+
+    git_repository(
+        name = "platforms",
+        tag = "0.0.5",
+        remote = "https://github.com/bazelbuild/platforms.git",
+    )
+
+    http_archive(
+        name = "glpk",
```

Comment:
GLPK is GPLv3 licensed, so I think we can't link it in without violating the license.